MarkdownView/markdownview/src/main/AndroidManifest.xml

```diff
@@ -1,11 +1,4 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="us.feras.mdv">
-
-    <application
-        android:allowBackup="true"
-        android:label="@string/app_name"
-        android:supportsRtl="true">
```

Comment:
Trying out this library to render markdown. This is indeed the case, and it is currently necessary to add `tools:replace="allowBackup, label"` to the `<activity>` element. Slightly annoying! Otherwise it seems to work fine.